lib/std/crypto/aes.zig

```diff
@@ -215,6 +240,29 @@ fn AESDecrypt(comptime keysize: usize) type {
         pub fn decrypt(ctx: Self, dst: []u8, src: []const u8) void {
             decryptBlock(ctx.dec[0..], dst, src);
         }
+
+        pub fn ige(ctx: Self, dst: []u8, src: []const u8, iv: [(keysize / 8) * 2]u8) void {
+            std.debug.assert(src.len >= dst.len);
+            
+            const b = keysize / 8;
+            var c: [b]u8 = undefined;
+            var m: [b]u8 = undefined;
+
+            std.mem.copy(u8, &c, iv[0..b]);
+            std.mem.copy(u8, &m, iv[b..]);
+
+            var o: usize = 0;
+            while (o < src.len) : (o += b) {
+                const t = src[o..o+b];
```

Comment:
Write as `const t = src[o..][0..b]` instead to get a comptime known length slice (pointer to array)
Add `const d = dst[o..][0..b]` as well?